lib/puppet/functions/simp_banners/fetch.rb

```diff
@@ -0,0 +1,77 @@
+# Retrieve a banner from the known list and format it according to the passed
+# options
+#
+Puppet::Functions.create_function(:'simp_banners::fetch') do
```

Comment:
simp_banners::select seems more intuitive than simp_banners::fetch?

Reply:
I would expect a method named `fetch` more likely to return something than one named `select`. 

(And yet there's `Hash.select`, so :man_shrugging: )